api/api_test.go

```diff
@@ -1110,6 +1111,79 @@ func TestServer_SendAction(t *testing.T) {
 	}
 }
 
+func TestServer_SendActionErrorCase(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	ctx := context.Background()
+
+	testTransfer, _ = testutil.SignedTransfer(identityset.Address(1).String(),
+		identityset.PrivateKey(2), 1, big.NewInt(10), []byte{}, 0,
+		big.NewInt(0))
+	tests := []struct {
+		server func() (*Server, error)
+		action *iotextypes.Action
+		err    string
+	}{
+		{
+			func() (*Server, error) {
+				cfg := newConfig(t)
+				return createServer(cfg, true)
```

Comment:
is it possible to move these 2 lines up-front, and use it as common var? so no need to have `server` in the struct